Firestore/Source/API/FIRFirestore.mm

```diff
@@ -171,7 +171,9 @@ - (void)setSettings:(FIRFirestoreSettings *)settings {
     _settings = settings;
     _firestore->set_settings([settings internalSettings]);
 
-#if HAVE_LIBDISPATCH
+#ifndef HAVE_LIBDISPATCH
+#error HAVE_LIBDISPATCH must be set to 1 or 0; verify the #include for config.h.
```

Comment:
This should be a string error message (i.e. enclose in double-quotes).

This is a lot of text. How about

```c
#error "Missing HAVE_LIBDISPATCH; please include config.h"
```